examples/hello_world.ts

```diff
@@ -1,5 +1,5 @@
 import { serve } from "https://deno.land/std@0.152.0/http/server.ts";
-import { router } from "https://deno.land/x/rutt/mod.ts";
```

Comment:
no, keep it this way, the import map fixes this for us that way the examples are kept the same as the actual code.